vital/algo/detected_object_set_output.h

```diff
@@ -110,6 +110,12 @@ class VITAL_ALGO_EXPORT detected_object_set_output
   virtual void write_set( const kwiver::vital::detected_object_set_sptr set,
                           std::string const& image_path ) = 0;
 
+  /// Perform end-of-stream actions.
+  /**
+   * This method writes any necessary final data to the currently open file.
+   */
+  virtual void complete() {}
```

Comment:
This might cause problems with two implementations of detected_object_set_output in arrows/core since neither arrows are overriding it.

Reply:
They shouldn't need to override it as they don't need to do anything special when the input ends. Or am I missing something?

Reply:
No you are correct.